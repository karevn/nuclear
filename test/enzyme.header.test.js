import React from 'react';
import { shallow } from 'enzyme';

import Header from '../app/components/Header';

describe('<Header />', () => {
  it('renders a header with no children', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
    expect(wrapper.containsMatchingElement(
      <div className='header_container' />
    ));
  });

  it('renders a header with children', () => {
    const wrapper = shallow(<Header>test</Header>);
    expect(wrapper).toMatchSnapshot();
    expect(wrapper.containsMatchingElement(
      <div className='header_container' />
    ));
  });
});
