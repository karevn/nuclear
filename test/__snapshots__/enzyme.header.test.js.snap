// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> renders a header with children 1`] = `
<div
  className="header_container"
>
  test
</div>
`;

exports[`<Header /> renders a header with no children 1`] = `
<div
  className="header_container"
/>
`;
